RAMP/unpacker.py

```diff
@@ -59,8 +62,16 @@ def _validate_zip_file(zip_file, max_bundle_size_mb):
     exceed a certain threshold.
     """
     infolist = zip_file.infolist()
-    # We're expecting exactly two files.
-    if len(infolist) != 2:
+    # We're expecting to have .so file, module.json and deps dir.
+    files_counter = 0
+
+    for file in infolist:
```

Comment:
have you tested that this goes recursively into the dirs?
I'm thinking what if the loop sees only 3 items (.so, .json, deps-dir)

Reply:
yes it works